analyzer/flow_server.go

```diff
@@ -81,7 +87,23 @@ func (c *FlowServerWebSocketConn) OnMessage(client shttp.WSSpeaker, m shttp.WSMe
 		logging.GetLogger().Errorf("Error while parsing flow: %s", err.Error())
 		return
 	}
-	logging.GetLogger().Debugf("New flow from Websocket connection: %+v", f)
+	//logging.GetLogger().Debugf("New flow from Websocket connection: %+v", f)
+	if len(c.ch) >= c.maxFlowBufferSize {
+		c.numOfLostFlows++
+		if c.timeOfLastLostFlowsLog.IsZero() ||
+			(c.timeOfLastLostFlowsLog.Sub(time.Now()) >= time.Second) {
+			if c.numOfLostFlows < c.maxFlowBufferSize/4 {
```

Comment:
Can you elaborate on why purging the chan will help instead of just skipping ? moreover you purge the chan only  when logging thus each second. I think that skipping new flow should be enough letting the "insert" goroutine consuming the chan purging it.

Reply:
I just skip now, you are right